assets/charts/control-plane/calico/templates/calico-kube-controllers.yaml

```diff
@@ -0,0 +1,50 @@
+# See https://github.com/projectcalico/kube-controllers
```

Comment:
Was this change on purpose?

Reply:
I mean moving from deployment.yaml to this file?

Reply:
No I mean , this makes the file naming correct to the resource it holds.

Reply:
the file was named `deployment.yaml`, it contain the deployment resource. I renamed it reflect better.
Similar with `daemonset.yaml`

deployment.yaml - calico-kube-controllers.yaml
daemonset.yaml - calico-node.yaml